package.json

```diff
@@ -74,6 +75,8 @@
     "inquirer": "^8.0.0",
     "moment": "^2.30.1",
     "solhint": "^5.0.3",
-    "typescript": "^5.7.2"
+    "typescript": "^5.7.2",
+    "@types/lodash.merge": "^4.6.9",
+    "@types/node": "^22.10.2"
```

Comment:
Package managers add packages in alphabetical order. Let's keep them at the top to avoid conflicts